diff --git a/generator/google/protobuf/__init__.py b/generator/google/protobuf/__init__.py
index 6210a40..82be65c 100644
--- a/generator/google/protobuf/__init__.py
+++ b/generator/google/protobuf/__init__.py
@@ -32,8 +32,3 @@
 
 __version__ = '3.0.0'
 
-if __name__ != '__main__':
-  try:
-    __import__('pkg_resources').declare_namespace(__name__)
-  except ImportError:
-    __path__ = __import__('pkgutil').extend_path(__path__, __name__)
-- 
2.18.0.345.g5c9ce644c3-goog

